test/functional/script/luacats_parser_spec.lua

```diff
@@ -1,4 +1,5 @@
-local t = require('test.functional.testutil')()
+local t = require('test.testutil')
```

Comment:
Since we no longer merge the modules, this makes it obvious that this test doesn't use a Nvim session.